app/controllers/meals_controller.rb

```diff
@@ -6,13 +6,16 @@ class MealsController < ApplicationController
   # GET /meals.json
   def index
     @meals = current_user.meals.includes(:recipe)
-    @meals_json = @meals
-      .collect{ |meal| {
-      title: meal.recipe.name,
-      start: meal.date.strftime("%Y-%m-%d"),
-      url: edit_meal_url(meal),
-      className: meal.time
-    } }.to_json
+    @meals_json = @meals.
+      sort_by(&:time_position).
+      map{ |meal|
```

Comment:
Avoid using {...} for multi-line blocks.<br>Align the operands of an expression in an assignment spanning multiple lines.